packages/auth/src/utils/rbac.ts

```diff
@@ -215,6 +199,22 @@ const getDeploymentScopes = async (id: string) => {
   ];
 };
 
+const getRunbookScopes = async (id: string) => {
```

Comment:
we already have this logic?

https://github.com/sizzldev/ctrlplane/blob/ba12569a384c755edbe8de69e47cadcb9e093292/packages/auth/src/utils/rbac.ts#L96